lang/csharp/src/apache/main/Generic/PreresolvingDatumReader.cs

```diff
@@ -120,18 +123,23 @@ private ReadItem ResolveReader(Schema writerSchema, Schema readerSchema)
             {
                 case Schema.Type.Null:
                     return ReadNull;
+
                 case Schema.Type.Boolean:
                     return ReadBoolean;
+
                 case Schema.Type.Int:
                     {
                         switch (readerSchema.Tag)
                         {
                             case Schema.Type.Long:
-                                return Read(d => (long) d.ReadInt());
+                                return Read(d => (long)d.ReadInt());
```

Comment:
personal opinion: it reads easier with the space between `(long)` and `d.`

Reply:
While I actually agree, it's just the pattern that is suggested that the cast appears to be modifying the value that follows.  At least with C# it may confuse people to have a space in-between.